app/mailers/course_update_email_mailer.rb

```diff
@@ -1,14 +1,16 @@
 class CourseUpdateEmailMailer < GovukNotifyRails::Mailer
-  def course_update_email(course, attribute_changed, user)
+  def course_update_email(course, updated_attribute_name, original_value, updated_value, user)
     set_template(Settings.govuk_notify.course_update_email_template_id)
 
     set_personalisation(
       provider_name: course.provider.provider_name,
       course_name: course.name,
       course_code: course.course_code,
-      attribute_changed: attribute_changed,
+      attribute_changed: I18n.t("course.update_email.#{updated_attribute_name}"),
       attribute_change_datetime: format_update_datetime(course.updated_at),
       course_url: create_course_url(course),
+      original_value: formatter.call(name: updated_attribute_name, value: original_value),
```

Comment:
I'm not massively familiar with Govuk Notify - am I right in thinking there is a template somewhere that needs to be updated to use`original_value` ?

Reply:
`set_personalisation` sets up the values for the placeholders in govuk notify template. I think what makes it hard to visualize is that the template is not stored in this repo but on govuk notify service itself.

Reply:
Yes, there's a template. I haven't updated it as yet. It currently looks like this

<img width="788" alt="Screenshot 2020-03-13 at 17 41 27" src="https://user-images.githubusercontent.com/5216/76645945-efe8f700-6551-11ea-895b-cbf1c3733137.png">